src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE books (
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

```

Comment:
```suggestion
id bigint  AUTO_INCREMENT PRIMARY KEY,
```
PRIMARY KEY - can't be null.
When creating a table in MySQL, use bigint column type for storing id.